tests/test_pluginmanager.py

```diff
@@ -45,6 +45,7 @@ def get_states():
 def test_disable_itself(mocker, dialog_content):
     mock = mocker.patch("tkinter.messagebox.askokcancel", return_value=False)
     dialog_content.treeview.selection_set(["pluginmanager"])
+    dialog_content.disable_button.state(["!disabled"])
```

Comment:
It might be better to run `_on_select()`, which sets the buttons enabled or disabled:

```suggestion
    dialog_content.treeview.event_generate("<<TreeviewSelect>>")
```